ui/app/AppMain.qml

```diff
@@ -800,26 +802,22 @@ Item {
             width: 350
             x: parent.width / 2 - width / 2
             y: parent.height / 2 - height / 2
-            // TODO improve this to work with community Chats as well
-            modelList: mainModule.getChatSectionModule().model
```

Comment:
@osmaczko I connected a comment here, but it's about `StatusInputListPopup`, so this `StatusInputListPopup` component is not something we're proud of, would be good to check with @PascalPrecht is there already any convenient component on the `StatusQ` side which may replace this one.

For this component we should just forward a model with expected props exposed and it should know how to do the rest, and it should have just a signal e.g. "itemClicked(id)" to notify outside world what's happened in there.

If we don't have such component we may improve this one a bit in that direction.

Reply:
Hi!

Missing the context here. What is a `StatusInputList`?

Reply:
@PascalPrecht it's a simple poup actually consists of two parts, top section which is a search box and list of items as a section below that search box. Typing a word in search box you're filter out the list and a click on the item from that list should emit a signal with id of the clicked item e.g. "itemClicked(item-id)". Do we have something similar on the `StatusQ` side. This is a current implementation we're using [here](https://github.com/status-im/status-desktop/blob/master/ui/imports/shared/status/StatusInputListPopup.qml). And [here](https://github.com/status-im/status-desktop/pull/4727#issue-840258910) is a video displaying its use cases.

Reply:
Ah I see!
No we don't have such a thing in StatusQ, however, it sounds like this is something that can/should be composed of existing StatusQ components (`StatusInput`, `StatusListItem` etc)

Reply:
Please see updated version which uses both StatusInput and StatusListItem -> https://github.com/status-im/status-desktop/pull/4727#issue-840258910

@PascalPrecht I've introduced new component for that: `ui/imports/shared/status/StatusSearchListPopup.qml`. I guess it should go to StatusQ, shouldn't it?